tests/run-images.sh

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
 set -ex
 
-ver=(NATS_SERVER 2.9.23)
```

Comment:
I think this one has to be `2.9.23` actually

Reply:
Hrm. Let me check, I think the tests run on which ever version is currently being released. At least that is how it worked before.

Reply:
There are only one set of tests for both versions so which is being released is the one that would replace this value. The 2.9.x one has not changed, so no need for testing that.

I believe you did a dual release before correct, so both versions needed to be tested?

Reply:
For posterity, noting @wallyqs and I spoke and no need to change since only one version bump is happening.